inner-scripts/kube-bind

```diff
@@ -78,15 +79,28 @@ else
     echo "kcp workspace $space_name's ID is $kcp_id"
 fi
 
-#TODO: move the starting of konnector here
-
 kubectl ws "root:$space_name"
 
 if kubectl get crd $resource_to_bind.edge.kubestellar.io &> /dev/null; then
     echo "CRDs for $resource_to_bind already in place"
     exit 0
 fi
 
+if [ "$start_konnector" == true ]
+then
+    if ! kubectl get namespace kube-system &> /dev/null
+    then kubectl create namespace kube-system &> /dev/null
+    fi
+    echo "starting kube-bind konnector for space $space_name in background"
+    konnector &> /dev/null &
+    sleep 2
```

Comment:
As a general rule, a fixed delay is bad because you never know how overloaded the node may be and thus how long you have to wait. A better approach is a loop that waits until the needed condition is met, possibly giving up after a long time.

Reply:
Yes. This issue is also related to another issue, which is there might be multiple konnectors, each for one space. I need to figure out which one is the one just started. Will address this in another PR.